qemu/tests/floppy.py

```diff
@@ -193,7 +191,7 @@ def test(self):
             self.srchost = params["hosts"][0]
             self.dsthost = params["hosts"][1]
             self.is_src = params["hostid"] == self.srchost
-            self.mig = self.mig_type(test, params, env, False, )
+            self.mig = self.mig_type(test, params, env, True, )
```

Comment:
@spcui   prepare_env = True , will create image file before VM create,  if set it to true,  how to ensure old guest os image not be recreate ?